### ISSUE TITLE
1.5.5 doesn't compile in XCode 10

### ISSUE THREAD

**Issue by cepages** [backtrace-labs/backtrace-cocoa#42]
Hi, I have a couple of computers one of them with XCode 10.

The one with XCode 11 works fine, but the one with XCode 10 won't compile due this two errors:

`Missing argument for parameter 'timestamps' in call` in `BacktraceApi`
```
        self.backtraceRateLimiter = BacktraceRateLimiter(reportsPerMin: reportsPerMin)
```
![image](https://user-images.githubusercontent.com/5259830/94162584-c6087980-fe7e-11ea-995f-f9a1163c9385.png)

and 

`Type 'JSONSerialization.ReadingOptions' has no member 'fragmentsAllowed'` in `private extension Optional where Wrapped == Data`
```
            return (try? JSONSerialization.jsonObject(with: data, options: [.fragmentsAllowed])) ?? ""
```

![image](https://user-images.githubusercontent.com/5259830/94162719-e801fc00-fe7e-11ea-9e77-0dd6bc432612.png)

Version `1.5.3` will work in XCode 10.

**Comment by krml19** [comment]
Hi @cepages, just released version `1.5.6`. Please give it a try to check if I can close this issue.

**Comment by cepages** [comment]
I tried in XCode 10 and XCode 11 and works fine. Thanks
